docs/web3-eth-contract.rst

```diff
@@ -72,8 +79,8 @@ Example
 .. code-block:: javascript
 
     const myContract = new web3.eth.Contract([...], '0xde0B295669a9FD93d5F28D9Ec85E40f4cb697BAe', {
-        from: '0x1234567890123456789012345678901234567891', // default from address
-        gasPrice: '20000000000' // default gas price in wei, 20 gwei in this case
+        defaultAccount: '0x1234567890123456789012345678901234567891', // default from address
```

Comment:
I think you should change the comment for defaultAcount.

It should be the **`default account address`** instead of **default from address**

Reply:
I think it's more understandable with ``// default from address``. Because it's in the end the from property of the transaction.

Reply:
okay cool.